synapse/lib/auth.py

```diff
@@ -1286,10 +1287,16 @@ async def setAdmin(self, admin, gateiden=None, logged=True):
     async def setLocked(self, locked, logged=True):
         if not isinstance(locked, bool):
             raise s_exc.BadArg(mesg='setLocked requires a boolean')
+
         if logged:
             await self.auth.setUserInfo(self.iden, 'locked', locked)
+            if not locked:
+                await self.auth.setUserInfo(self.iden, 'policy:attempts', 0)
```

Comment:
Lets discuss this, since it feels like we're conflating any `locked` status change with "clearing password attempts".